### ISSUE TITLE
Package.json not found Cordova 7.0

### ISSUE THREAD

**Issue by gaurav-vachhani** [katzer/cordova-plugin-app-event#12]
Facing issue with this plugin where package.json file not found while compiling with Cordova 7.0.

Error: Failed to fetch plugin https://github.com/katzer/cordova-plugin-local-notifications.git via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 194 Error output:
npm ERR! path /Adreno Projects/Estays Project/Estays/node_modules/cordova-plugin-app-event/package.json
npm ERR! code ELOOP
npm ERR! errno -62
npm ERR! syscall open
npm ERR! ELOOP: too many symbolic links encountered, open '/Adreno Projects/Estays Project/Estays/node_modules/cordova-plugin-app-event/package.json'

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Vachhani/.npm/_logs/2017-07-19T07_32_57_641Z-debug.log

**Comment by fizdalf** [comment]
I have the same issue, first symbolic links, and then if I remove the symbolic links it just says 
```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules\cordova-plugin-app-event" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
```

**Comment by ghost** [comment]
@fizdalf @gaurav-vachhani 
did you guys solve your problems or do you need any further help?

**Comment by fizdalf** [comment]
I had to use a fork of this repo to make it work, 

https://github.com/martinsimonovskipc/cordova-plugin-app-event that sorts the issue out.

Want me to make a pull request with that?

**Comment by katzer** [comment]
I will update the package.json file and push to NPM.

BTW with the next version of the local-notification plugin the cordova-plugin-app-event plugin isn't required anymore. See branch x.

**Comment by katzer** [comment]
See 96cbbab. Have released v1.2.1 on NPM.

**Comment by Seanny123** [comment]
I'm also getting the same error, however I was able to add branch `x` of `cordova-plugin-local-notifications` by running:

```
rm -r node_modules/cordova-plugin-app-event
cordova plugin add https://github.com/katzer/cordova-plugin-local-notifications#x
```

**Comment by gaurav-vachhani** [comment]
@ahermle93 I have solved this issue after installing the 1.2.1 version of cordova-plugin-app-event. 

If anyone else face this same problem, please install the specific (1.2.1) version of cordova-plugin-app-event and then cordova-plugin-local-notifications.

**Comment by FatehiAlqadasi** [comment]
@gaurav-vachhani thank you
